tests/perfsys.ts

```diff
@@ -20,29 +20,9 @@ module perftest {
     export var getCurrentDirectory = ts.sys.getCurrentDirectory;
     var exit = ts.sys.exit;
 
-    var args = ts.sys.args;

```

Comment:
Can you explain why these changes were necessary? I'm personally not that familiar with the perfsys.ts stuff